measurements/api/legacy_explorer.py

```diff
@@ -0,0 +1,135 @@
+'''
+prefix: /api/legacy_explorer
+
+That's legacy API for old version of OONI Explorer that mimics (and replaces)
+API at http://explorer.ooni.torproject.org/api/. That's a hack, don't use it.
+This API will eventually be refactored & split across private.py and public API.
+'''
+
+import json
+import itertools
+
+from flask import Blueprint, current_app, request
+from flask.json import jsonify
+from sqlalchemy import func # , or_, and_, false, true
+
+from measurements.models import Report, Input, Measurement, Autoclaved
+from measurements.api.measurements import fetch_autoclaved_text
+
+# prefix: /api/legacy_explorer
+api_legacy_explorer_blueprint = Blueprint('api_legacy_explorer', 'measurements')
```

Comment:
I would put these legacy endpoint inside of the private blueprint (which has the `_` prefix) and maybe call it `legacy-explorer` for consistency.

Reply:
I've put it under separate prefix for nginx & separate blueprint to `git rm` single file when explorer becomes "revamped". What's the drawback of this way?